src/plopp/backends/matplotlib/image.py

```diff
@@ -132,29 +133,6 @@ def __init__(
         if need_grid:
             self._ax.grid(True)
 
-        # Cache slicing order for hover values
```

Comment:
Is this related to datetimes?

Reply:
Kind of, it was the old way of getting the values for the fomatter. This is where the units for each axis were used from, but now, we instead just use the units that are stored in the canvas. The info was basically stored in both places and I think it makes more sense to just have it on the canvas.

We basically need to slice the image data to get the z value. We use label-based slicing, and for that we need the dim associated to each axis, and the unit. The canvas knows both these things, and the canvas now just sends a tuple `(dim, sc.scalar(value, unit=unit)` to the `format_coord` which is used as is for slicing.